cpp/srs_db/grumpkin/monomial/README.md

```diff
@@ -0,0 +1,16 @@
+# Quick-and-dirty Grumpkin transcript
+
+The Grumpkin transcript currently departs in structure from the BN254
+transcript in that:
+ - It does not contain a checksum
+ - It does not contain any g2 points (indeed, there is no grumpkin::g2).
+ - The transcript generation binary only produces a single transcript file.
+   If more than 504000 points are desired at some point, it is likely we will 
+   need to a small refactor so that more files are created.
```

Comment:
"to do"?

Reply:
I don't know if we ever actually need to address this, but yeah I'll make an issue nonetheless.

Reply:
#492